app/src/main/java/com/truongdc21/quickquotes/Presenter/FragmentSearch/SearchFragmentContact.kt

```diff
@@ -1,4 +1,21 @@
 package com.truongdc21.quickquotes.presenter.fragmentSearch
 
+import android.view.View
+import com.truongdc21.quickquotes.data.model.Search
+import com.truongdc21.quickquotes.utils.base.BasePresenter
+
 interface SearchFragmentContact {
+
+    interface Presenter: BasePresenter<View>{
+        fun getListSearchAPI()
+        fun getListSearchHistory()
+        fun insertSearchHistory(search: Search)
+        fun deleteSearchHistory(id : Int)
+    }
+    interface View {
```

Comment:
```suggestion

    interface View {
```
add 1 blank line between 2 class